### ISSUE TITLE
pacmog advertises that it works with no_std out of the box, but doesn't declare no_std.

### ISSUE THREAD

**Issue by kevinclark** [AkiyukiOkayasu/pacmog#42]
Here's my first error messages:

```
❯ cargo build | less
   Compiling pacmog v0.4.1
error[E0463]: can't find crate for `std`
  |
  = note: the `thumbv6m-none-eabi` target may not support the standard library
  = note: `std` is required by `pacmog` because it does not declare `#![no_std]`

error: cannot find attribute `derive` in this scope
 --> /Users/kev/.cargo/registry/src/index.crates.io-6f17d22bba15001f/pacmog-0.4.1/src/aiff.rs:9:3
  |
9 | #[derive(Debug, PartialEq, Default)]
  |   ^^^^^^
  |
help: consider importing this attribute macro
  |
1 + use core::prelude::rust_2024::derive;
 ```

**Comment by AkiyukiOkayasu** [comment]
Thanks for reporting it. The crate was intended to be used with a Raspberry Pi Pico, but I was not aware of this problem as I no longer needed it.
I believe this has been fixed in v0.4.2. Please give it a try.

**Comment by kevinclark** [comment]
Got it. Looks like it should do the trick, thanks!

Next issue I've run into is that the crate seems to be no-alloc except for the anyhow usage. I'd like to fix that - I'm thinking I'll replace the anyhow usage with [embedded_error_chain](https://crates.io/crates/embedded-error-chain). Is that a change you'd take? Is no-alloc a goal at all? Got a better idea?
